Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Basic docker image for Monocle
+# Usage:
+#   docker build -t Monocle
+#   docker run -d --name Monocle -P Monocle
+
+FROM python:3.6
```

Comment:
Can add it in the description but I guess people using docker know they need to provide a persistent working directory on the local filesystem if they want anything persistent... so indeed I also pass a -v option with a working directory for pickles, logs, ... That's more something we would need to describe in a wiki than in the dockerfile itself.